src/lib.rs

```diff
@@ -163,6 +272,47 @@ enum Register {
     INT1_DURATION = 0x38,
 }
 
+/// Output Data Rate
+#[derive(Debug, Clone, Copy)]
+pub enum ODR {
+    /// 95 Hz data rate
+    Hz95  = 0x00,
```

Comment:
I like this way of writing the unit before the number since variant names can't start with a digit. 👍